pkg/topology/full/full.go

```diff
@@ -155,6 +157,28 @@ func (d *driver) ClosestPeer(addr swarm.Address) (swarm.Address, error) {
 	return closest, nil
 }
 
+func (d *driver) Connected(ctx context.Context, addr swarm.Address) error {
+	return d.AddPeer(ctx, addr)
+}
+
+func (d *driver) Disconnected(swarm.Address) {
+	panic("todo")
```

Comment:
since there are no tests covering this in the `full` driver. I have replaced the `panic` with a todo, since this was never implemented as part of the `full` driver i don't wanna keep cramming more functionality and tests into this PR. moreover this might be a waste of time to pour more engineering hours over this component. so let's go with a todo and if we really need this further down the road we can implement this quickly